features/cellular/framework/device/CellularStateMachine.cpp

```diff
@@ -648,6 +648,14 @@ void CellularStateMachine::send_event_cb(cellular_connection_status_t status)
     }
 }
 
+void CellularStateMachine::change_timeout(const int &timeout)
+{
+    _cb_data.status_data = _current_event;
```

Comment:
Maybe we should add here the description why the original solution did not work and this was needed.

Reply:
fixed